src/components/o-modal/index.scss

```diff
@@ -0,0 +1,68 @@
+@import '../../styles/mixins/respond';
+
+$modal-background-color: #fff;
+$close-button-color: #00008f;
+$close-button-color-hovered: #000;
+
+.m-modal {
+  position: fixed;
+  left: 0;
+  top: 0;
+
+  display: block;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
+}
+
+.m-modal--hidden {
+  display: none;
+}
+
+.m-modal__content {
+  margin: auto;
+  overflow: scroll;
+
+  padding: 20px;
+  width: 100%;
+  height: 100%;
+  background-color: $modal-background-color;
+
+  border: 1px solid #888;
+  text-align: center;
+
+  @include respond-up(md) {
+    margin-top: 20vh;
+    width: 80vw;
+    height: 60vh;
+  }
+
+  @include respond-up(lg) {
+    margin-top: 20vh;
+    width: 70vw;
+    height: 60vh;
+  }
+}
+
+.m-modal__close-button {
+  float: right;
+  color: $close-button-color;
+}
+
+.m-modal__close-button:hover,
+.m-modal__close-button:focus {
+  color: $close-button-color-hovered;
+}
+
+/* Override browser button styles */
+button.m-modal__close-button {
+  background-color: transparent;
+  border: none;
+  font-family: inherit;
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  appearance: none;
+  cursor: pointer;
```

Comment:
do we need cursor pointer for a button?

Reply:
this is the default. the question would imho rather be, if we do not need it? the styleguide does not mention it, the native browser element has it.

Reply:
I just found out that there is a sass file I can import that is doing exactly what I did here. I will just import that. In there, the cursor is also set to pointer.